src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java

```diff
@@ -126,6 +126,14 @@ public ContentPackage2FeatureModelConverter(boolean strictValidation) {
         }
     }
 
+    public EntryHandlersManager getHandlersManager() {
```

Comment:
@kwin - I guess I'm not sure if it is a good idea to expose the entry handlers. Is that really necessary? 

It seems like what we really want is to be able to generate a new content package and then have it processed like a normal content package would be. At any rate, the current handling has a bug - see my next comment in the BundleEntryHandler

Reply:
Hmm, for me that sounds overly complex. Especially as that would mean processing Sling-Initial-Content would need the following phases:
1. Extract content into 2 content packages
2. Process the 2 content packages again and extract into feature model

That would mean a lot of overhead.Is there any particular issue you see with the recursive approach inside the `BundleEntryHandler`?

Reply:
Yeah, it would be overhead - the benefit would be that we don't have a second code path hidden inside the handler. I guess I don't right now see an issue with only calling the handlers again - however, that could obviously change in the future (not sure if it would be possible for a bundle to have initial content that could be picked up by the installer again...). 

Could we maybe move the processing to the Converter so that at least it is more in the open what is going on? Something like:

```
public void process(@NotNull String path, Callable<Archive> archiveCallable)
```

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/74/commits/ce8e2cf97d8cb99331d74f4bd6b2a52529efd0ce.